kubernetes/create-job-dependencies.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash -e
 
-kubectl create --filename secrets/db-secrets.yml
+db_name=$1
+
+kubectl create configmap database-settings                                    \
```

Comment:
Please rename to `database-job-settings`.

Reply:
Done.